### ISSUE TITLE
Email fetching broken in osTicket-v1.10-RC1

### ISSUE THREAD

**Issue by pkgdemon** [osTicket/osTicket-1.8#2418]
Running the following command manually:

<code>/usr/local/bin/php /usr/local/www/apache24/data/api/cron.php

Produces the error below:

<code>PHP Warning:  DateTimeZone::getOffset() expects parameter 1 to be DateTimeInterface, boolean given in /usr/local/www/apache24/data/include/class.format.php on line 520
PHP Fatal error:  Call to a member function getTimestamp() on boolean in /usr/local/www/apache24/data/include/class.format.php on line 521

This error was produced in test environment that was upgraded from 1.9.11.

**Comment by greezybacon** [comment]
This error alone is not breaking email fetching. Are you having any other issues you could describe?

**Comment by pkgdemon** [comment]
I did try to go through, and re-enter my info for the imap fetching section.  When I entered in the settings, and saved there were no errors.  That's why I thought the above command that executes cron.php may have been indicating why it wasn't working as this is what I have cron run to constantly pull in emails.  

Is there another command I can try to fetch email, or is there something else I can try to provide more information?  I've checked system, and there were  no errors with php components, etc.  There were some database upgrade errors stating that certain tables could not be removed during the upgrade.  Otherwise it was reported that the database was upgraded successfully.  I skimmed through them, and I'm not sure those would be helpful but I can post them if you like.  

There was a scripts folder included in the download that I wasn't sure what to do with?  Is that something I need to install somewhere?  I couldn't find any docs explaining what they were for.
